src/db/migration-helpers.ts

```diff
@@ -139,6 +139,15 @@ export async function updateViews(knex: Knex){
 
   // create views
   for (const table of tables) {
+
+    // return early if table doesn't exist
```

Comment:
To get us closer towards using migrations properly, this was added to make it more robust to new values being added in `db/tables`